bin/catalina.bat

```diff
@@ -243,7 +243,7 @@ if not "%LOGGING_MANAGER%" == "" goto noJuliManager
 set LOGGING_MANAGER=-Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager
 :noJuliManager
 
-rem Configure JAVA 9 specific start-up parameters
+rem Configure JAVA 9 specific start-up parameters - KEEP in line with service.bat
```

Comment:
What is this acronym "KEEP"?

Reply:
@michael-o It is intended as verb to "keep" those settings maintained in line with the settings in service.bat. I made it upper case to catch eyes. If it raises irritating by associating with an unknown acronym I could change it e.g. to
`Configure JAVA 9 specific start-up parameters - ensure to keep it in line with service.bat` or any other proposal from your end.

Reply:
Proposal is fine.

Reply:
> Proposal is fine.

You mean I should change it as I did propose or is it fine as it is?

Reply:
Change it.

Reply:
Done